cli/pull.go

```diff
@@ -34,6 +42,175 @@ func (cli *DogestryCli) CmdPull(args ...string) error {
 
 	cli.Config.SetS3URL(S3URL)
 
+	// Perform regular pull if we are explicitly told to _not_ use dogestry server(s)
+	if cli.Config.Slow {
```

Comment:
Should this config option not be `-slow` but rather something like `-force-local`?